server/src/main/resources/db/changelog/20170214115606-uber-cert-rewrite.xml

```diff
@@ -0,0 +1,203 @@
+        name="delete_pool_source_sub_rewrite"
+        value="delete ss from cp2_pool_source_sub ss
+                   inner join cp_pool p on ss.pool_id = p.id
+                   inner join cp2_products prod ON prod.uuid = p.product_uuid
+               where prod.name like '%_ueber_product';" />
+
+    <!-- Delete any ueber pools -->
+    <property
+        dbms="mysql"
+        name="delete_ueber_pools_rewrite"
+        value="delete p from cp_pool p
+                   inner join cp2_products prod ON prod.uuid = p.product_uuid
+                   where prod.name like '%_ueber_product';"/>
+
+    <!-- Delete any ueber owner content  -->
+    <property
+        dbms="mysql"
+        name="delete_owner_content_rewrite"
+        value="delete oc from cp2_owner_content oc
+                   inner join cp2_product_content pc on pc.content_uuid=oc.content_uuid
+                   inner join cp2_products p on p.uuid=pc.product_uuid
+                   where p.name LIKE '%ueber_product';"/>
+
+    <!-- Delete any ueber content  -->
+    <property
+        dbms="mysql"
+        name="delete_ueber_content_rewrite"
+        value="delete c from cp2_content c
+                   inner join cp2_product_content pc on pc.content_uuid=c.uuid
+                   inner join cp2_products p on p.uuid=pc.product_uuid
+                   where p.name LIKE '%ueber_product';"/>
+
+    <!-- Delete any ueber owner products  -->
+    <property
+        dbms="mysql"
+        name="delete_owner_products_rewrite"
+        value="delete op from cp2_owner_products op
+                   inner join cp2_products p on op.product_uuid = p.uuid
+                   where name like '%ueber_product';"/>
+
+    <!-- Delete any ueber products -->
+    <property
+        dbms="mysql"
+        name="delete_ueber_products_rewrite"
+        value="delete p from cp2_products p where name like '%_ueber_product';"/>
+
+    <!-- Delete any ueber consumers -->
+    <property
+        dbms="mysql"
+        name="delete_ueber_consumers_rewrite"
+        value="delete c from cp_consumer c where name like 'ueber_cert_consumer';"/>
+
+    <!-- Delete the uebercert consumer type as it is no longer needed. -->
+    <property
+        dbms="mysql"
+        name="delete_ueber_consumer_type"
+        value="delete t from cp_consumer_type t where t.label='uebercert';" />
+
+    <changeSet id="20170214115606-1" author="mstead">
+        <comment>Cleanup all ueber data so that the new style ueber certs can be used.</comment>
+        <sql>${delete_ueber_certs_rewrite}</sql>
+        <sql>${delete_pool_source_sub_rewrite}</sql>
+        <sql>${delete_ueber_ents_rewrite}</sql>
+        <sql>${delete_ueber_pools_rewrite}</sql>
+        <sql>${delete_owner_content_rewrite}</sql>
+        <sql>${delete_ueber_content_rewrite}</sql>
+        <sql>${delete_owner_products_rewrite}</sql>
+        <sql>${delete_ueber_products_rewrite}</sql>
+        <sql>${delete_ueber_consumers_rewrite}</sql>
```

Comment:
This is an interesting way of doing this. Is there a reason you opted to do these queries as properties rather than changesets (or completely different changelog files for each db)?